src/Form/MessageTypeForm.php

```diff
@@ -130,6 +130,13 @@ public function form(array $form, FormStateInterface $form_state) {
       '#states' => $states,
     ];
 
+    $form['settings']['token options']['token replace'] = array(
+      '#type' => 'checkbox',
+      '#title' => t('Token replace'),
+      '#description' => t('When this option is selected, token processing will happen.'),
+      '#default_value' => isset($settings['token options']['token replace']) ? !empty($settings['token options']['token replace']) : TRUE,
```

Comment:
the `!empty()` seems wrong, it should probably be:

`'#default_value' =>  !isset($settings['token options']['token replace']) || !empty($settings['token options']['token replace'])`